### ISSUE TITLE
Not being able to compile/run tests anymore

### ISSUE THREAD

**Issue by MiyamotoAkira** [exercism/xclojure#86]
With the change normalize test changes, where the (load-file) directive has been taken away, I cannot compile/run test anymore.
When using 'lein test' the following issue happens:

    Exception in thread "main" java.lang.RuntimeException: No such var: etl/transform, compiling:(etl_test.clj:6:10)

The etl.clj file has been created, the namespace is there as well as the transform function.

I would have expected that the lines that you added to the project.clj file

     :source-paths [""]
     :test-paths [""]

would set up the tests, but is not the case.

Trying to add a require doesn't help either. Only adding back the (load-file) seems to work.

**Comment by yurrriq** [comment]
Hmm... Can you help me reproduce your issue?

What did you try that led you to the following conclusion?

> Trying to add a require doesn't help either.

=====

Here's a little info about my personal setup, which may be slightly different from xclojure.

My `etl` dir looks like this:
```
etl
├── README.md
├── project.clj
├── src
│   └── etl.clj
└── test
    └── etl_test.clj
```

As such, my `project.clj` does **not** have:
```clojure
  :source-paths [""]
  :test-paths   [""]
```

My `test/etl_test.clj` starts like this:
```clojure
(ns etl-test
  (:require [clojure.core.typed :refer [check-ns]]
            [clojure.test :refer :all]
            etl)) ; <= Apparently, I'm requiring etl
```

and ends like this:
```cli
(deftest type-annotations
  (is (check-ns 'etl)))
```

Everything between is identical to xclojure.

**Comment by yurrriq** [comment]
I've checked out the xclojure repo and had no trouble running the tests agains the example solution.

```sh
ln -s example.clj etl.clj
```

```
etl
├── etl.clj -> example.clj
├── etl_test.clj
├── example.clj
└── project.clj
```


project.clj:
```clojure
  :source-paths [""]
  :test-paths [""]
```

etl_test.clj
```clojure
(ns etl-test
  (:require [clojure.test :refer :all]))
```

**Comment by yurrriq** [comment]
Will you please send me (a link to) your code? Maybe I can spot the issue.

**Comment by MiyamotoAkira** [comment]
Sorry for the delay. I will close the issue. The issue is not directly related to xclojure. Seems that Cider 0.9.1 has issues locating the file with the code. Happened on both Leap and Etl. But I only tried Etl outside of Cider. And there was a false negative in there. Once I saw your post and tried again I was able to eliminate the false negative. Tried installing Cider 0.10.0 and seems to work.
